src/rules.js

```diff
@@ -2,25 +2,27 @@
 
 var fs = require('fs');
 var path = require('path');
-var rules; // Cashing list of rules, so that we only load them once
```

Comment:
I had to remove the caching here, because multiple invocations with different `rulesdir` parameters will yield a different result for available rules. As gherkin-lint is mainly used on CLI, I think it's not as expensive. Currently the rules get loaded once, for checking the configuration and a second time when running them. This could be fixed by changing the `getConfiguration` method.